odoo-shippable/scripts/fix-vim-snippet.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+
+import os
+import glob
+import argparse
+
+
+class FixVimSnippet(object):
+
+    def __init__(self, extensions):
+        self.extensions = extensions
+        self.bundle_dir = os.path.join(os.environ.get('HOME'), '.vim',
+                                       'bundle')
+        self.snippet = {}
+        self.extension_snippet = {}
+        self.repeated_snippet = {}
+        self._fix()
+
+    def _fix(self):
+        if os.path.isdir(self.bundle_dir):
+            module = None
+            for path in glob.glob(self.bundle_dir + '/*/*/*.snippets'):
+                module, folder, file_name = (path.replace(
+                    self.bundle_dir + os.sep, '').split(os.sep))
+                name, extension = os.path.splitext(file_name)
+                if self.extensions and not name in self.extensions:
+                    continue
+                if not self.snippet.has_key(name):
+                    self.snippet[name] = []
+                self.snippet[name].append({'snippet': file_name,
+                                           'module': module,
+                                           'folder': folder,
+                                           'path': path})
+            for extension, snippets in self.snippet.iteritems():
+                if len(snippets) > 1:
+                    for snippet in snippets:
+                        if not self.extension_snippet.has_key(extension):
+                            self.extension_snippet[extension] = {}
+                        path = snippet['path']
+                        with open(path) as _file:
```

Comment:
`_` is a bad name if is not a private variable then don't use private name
Maybe, you could use `fobj`